generators/app/templates/gulp/tasks/vendor.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+var bundleVendorTask = function (gulp, plugins, config) {
+  var fileListPath = path.join(config.src.base, 'scripts/vendor.json');
+  var getFileList = function () {
+    return JSON.parse( fs.readFileSync(fileListPath) );
```

Comment:
I assumed those errors should be thrown enforcing existence of `vendor.json` with valid JSON inside as it's better than silently failing. I considered guarding from errors here but decided it's better they're reported. An example of how it looks now:

![zrzut ekranu 2017-05-19 09 57 45](https://cloud.githubusercontent.com/assets/1224792/26238553/d7ac81ec-3c79-11e7-91a6-df6dd46278cb.png)

Could go with try catch. What would be possible benefits?